### ISSUE TITLE
[Docs] Remove Extra space in Prerequisites

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#5496]
#### Current State
<img width="847" alt="Screen Shot 2022-05-09 at 2 28 18 PM" src="https://user-images.githubusercontent.com/7570704/167483384-058677b9-301e-4d06-b0d2-e9c97160ce09.png">

#### Desired State

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by karnatisrinivas** [comment]
Can I work on this @leecalcote ?

**Comment by KrishAryan** [comment]
Can I work on this @leecalcote as this will be my first time contributing.

**Comment by karnatisrinivas** [comment]
Hello @Akshit42-hue , I am unassigning myself from this issue. Kindly assign it to Krish.

**Comment by Akshit42-hue** [comment]
Sure @karnatisrinivas, go ahead @KrishAryan

**Comment by KrishAryan** [comment]
@leecalcote I observed in the HTML file that the space is due to style given in a CSS file to h5 tag and even inline css is not working over it. So my proposal is to replace h5 tag with strong tag.

**Comment by KrishAryan** [comment]
![screenshot (37)](https://user-images.githubusercontent.com/64402456/169400259-be109d78-b1bf-47f1-9ce7-f013a53ff063.png)

**Comment by leecalcote** [comment]
I'll let @karnatisrinivas offer feedback.

**Comment by karnatisrinivas** [comment]
Hello @KrishAryan. Even I feel like this space is due to the Styling applied to Tags. <strong> element also looks good.

**Comment by karnatisrinivas** [comment]
I think this might be the cause, but would you check if this? (docs/_site/assets/css/scss.css)
<img width="1440" alt="Screenshot 2022-05-20 at 11 58 14 AM" src="https://user-images.githubusercontent.com/52213014/169466991-41e95ed8-d8b1-4c74-a9ac-2bddb2afc188.png">

**Comment by asubedy** [comment]
Yes @KrishAryan, you can make one, we can review your work on the PR itself.
